src/theme/default_theme.rs

```diff
@@ -66,6 +66,17 @@ impl UiStyles {
                 conflicted:  Red.normal(),
             },
 
+            security_context: SecurityContext {
+                none:       Style::default(),
+                selinux: SELinuxContext {
+                    colon: Style::default().dimmed(),
+                    user:  Blue.normal(),
+                    role:  Green.normal(),
+                    typ:   Yellow.normal(),
+                    range: Cyan.normal(),
+                },
+            },
```

Comment:
How did you pick these colors? I think it’s better to use the same colors and in general to respect the conventions of other tools because these conventions were probably better considered and it makes exa consistent with them.

Reply:
I am not aware of any *standard* color scheme for SELinux contexts, so I picked colors such that the outputs looks sane.
Any suggestions how to improve the suggested coloring is welcome.

Reply:
Well then there’s no reason to change this now, it could always be improved later.